fdbserver/workloads/GetMappedRange.actor.cpp

```diff
@@ -155,7 +156,10 @@ struct GetMappedRangeWorkload : ApiWorkload {
 			auto& getRange = std::get<GetRangeReqAndResultRef>(it->reqAndResult);
 			auto& rangeResult = getRange.result;
 			//					std::cout << "rangeResult.size()=" << rangeResult.size() << std::endl;
-			ASSERT(rangeResult.more == false);
+			if (rangeResult.more) {
+				// Retry if the underlying request is not fully completed.
+				return true;
```

Comment:
What if retry still returns partial results, i.e., `rangeResult.more` is true? A better solution might be combining partial results together and then validate.

Reply:
Yes, eventually it should be better doing that way. But it's tricky to do that today because note that this is the is not the original range query but one of the underlying fan-out queries. A continuation will mean start a total different query. For now, we are focus on the performance of the happy path.

Reply:
OK. Can you add a TODO comment here.

Reply:
Added.